python/alibiexplainer/alibiexplainer/anchor_text.py

```diff
@@ -31,17 +31,20 @@ def __init__(self, predict_fn: Callable, explainer: alibi.explainers.AnchorText,
         self.predict_fn = predict_fn
         self.kwargs = kwargs
         logging.info("Anchor Text args %s", self.kwargs)
+        logging.info("Hey")
         if explainer is None:
             logging.info("Loading Spacy Language model for %s", spacy_language_model)
             spacy_model(model=spacy_language_model)
             self.nlp = spacy.load(spacy_language_model)
             logging.info("Language model loaded")
         self.anchors_text = explainer
 
-    def explain(self, inputs: List) -> Dict:
+    def explain(self, inputs: List, requestArgs: Dict) -> Dict:
```

Comment:
Is it cleaner to just pass in the entire request into explain? That's what the subclassed explain interface does.